scripts/core/editor3/reducers/table.jsx

```diff
@@ -162,10 +162,9 @@ const processCells = (state, fn) => {
         data: fn(cells, numCols, numRows, i, j, withHeader)
     });
```

Comment:
Let's do the same here:
```js
const entityDataHasChanged = true;
```
And use it as the last param to `onChange`.

Reply:
fixed